frontend/src/App.vue

```diff
@@ -33,6 +33,11 @@ export default {
     profile () {
       return this.api.get().profile()
     },
+    deploymentTime () {
+      const timestamp = window.environment.DEPLOYMENT_TIME || ''
```

Comment:
You check in the next line anyway
```suggestion
      const timestamp = window.environment.DEPLOYMENT_TIME
```